commands/snapshotdeletecommand.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+namespace PBST\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use ProfitBricksApi\ProfitBricksApi;
+use Exception;
+
+/**
+ * Class SnapshotDeleteCommand
+ *
+ * This command deletes snapshot(s) at ProfitBricks by snapshot ID or filtered by date (deletes older than).
+ */
+class SnapshotDeleteCommand extends Command
+{
+    private $config;
+    private $before;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->before = new \DateTime();
```

Comment:
The command option can be let **before** but to make the code more clear the variable name here should be `deleteBeforeTimestamp`.